### PR TITLE
Add help text about hover states to a11y how-to-review page

### PR DESCRIPTION
**Pull request by joelhsmith**
What's changed, or what was fixed?
- added help text about checking for hover states in the how to do an assessment page

**Fixes:** https://github.com/GoogleChrome/lighthouse/issues/5667

**Target Live Date:** Next release

- [x] This has been reviewed and approved by @joelhsmith
- [x] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label. (I do not think I have permissions to add labels here)
- [X] I've staged the site and manually verified that my content displays correctly.

**CC:** @robdodson

### CODE REVIEW

File: src/content/en/fundamentals/accessibility/how-to-review.md [hunk 1]
```diff
@@ -167,6 +167,16 @@ sure it’s truly removed from the keyboard flow. The proposed [inert
 attribute](https://github.com/WICG/inert) aims to make this easier by combining
 the behavior of both attributes.
 
+## Interactive elements like links and buttons should indicate their purpose and state
+
+Providing visual hints about what an element does helps people operate and navigate a site or app. In user experience this is called affordance. Providing affordance makes it possible for people to use a site on a wide variety of devices.
```
Comment (robdodson) [inline src/content/en/fundamentals/accessibility/how-to-review.md]: a few nits:

**Comment by petele** [comment]
Also note - this fails the build because `{# wf_blink_components: (something) #}` is missing from the top of the doc. @robdodson what should it be?

File: src/content/en/fundamentals/accessibility/how-to-review.md [hunk 2]
```diff
@@ -5,6 +5,7 @@ description: How to review your site for accessibility issues.
 
 {# wf_updated_on: 2017-07-24 #}
 {# wf_published_on: 2017-03-12 #}
+{# wf_blink_components: Blink>Accessibility #}
```
Comment (petele) [inline src/content/en/fundamentals/accessibility/how-to-review.md]: Awesome - thanks for adding.

**Comment by petele** [comment]
@robdodson - the correct tag would be `{# wf_blink_components: Blink>Accessibility #}`, which as added. thanks!

**Pull request merged**